dbt/adapters/dremio/credentials.py

```diff
@@ -20,34 +20,40 @@
 
 @dataclass
 class DremioCredentials(Credentials):
-    environment: Optional[str]
-    database: Optional[str]
-    schema: Optional[str]
-    datalake: Optional[str]
-    root_path: Optional[str]
+    environment: Optional[str] = None
+    UID: Optional[str] = None
+    PWD: Optional[str] = None
+    pat: Optional[str] = None
+    object_storage_source: Optional[str] = None
+    datalake: Optional[str] = None
```

Comment:
We don't need to specify the aliases here if we're going to specify them under _ALIASES. Datalake, root_path, database, and schema can all be removed as instance variables.

Reply:
Actually, it is the other way around. We can remove the new instance variables.